webapp/channels/src/components/post/post_options.tsx

```diff
@@ -235,10 +246,11 @@ const PostOptions = (props: Props): JSX.Element => {
     } else if (props.location === Locations.SEARCH) {
         const hasCRTFooter = props.collapsedThreadsEnabled && !post.root_id && (post.reply_count > 0 || post.is_following);
         options = (
-            <div className='col__controls post-menu'>
+            <ul className='col__controls post-menu'>
```

Comment:
[`<menu>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu#html) might be more appropriate, but 0/5 as either works fine.